app.js

```diff
@@ -1,11 +1,157 @@
+const cardObject = cardDetails;
+const mainWindow = document.querySelector('.mainWindow');
+const popupCard = document.querySelector('.popupContainer');
+const cardContainer = document.querySelector('.card-container');
+const workDetails = document.querySelector('.worklist');
+const techDetails = document.querySelector('.technologiesContainer');
+
+// popup function
+const popupmethod = (projectId) => {
+    let carditems = {};
+    
+
```

Comment:
- [ ] As requested by the previous reviewer, Please stick to the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) design as much as possible and fix the below issues and make sure your design matches with the Template.
 - Please increase the model width and top margin.

### Your design

![image](https://user-images.githubusercontent.com/83803414/168998202-79a6a217-9f0a-4925-922b-8910b2e445cf.png)

### Figma design

![image](https://user-images.githubusercontent.com/83803414/168998333-fa84942b-59f2-4e16-9b6f-4015533b4956.png)


  - Please use the correct color for project tags.

### Your design

![image](https://user-images.githubusercontent.com/83803414/168998609-6cf0630d-703e-4dd5-b0e6-9d57690feece.png)

### Figma design

![image](https://user-images.githubusercontent.com/83803414/168998571-36a709bf-ab62-4ce1-be3b-a00f6bf0b8ee.png)

  - Please remove boldness from the project description text and use the correct font weight and font size.

### Your design

![image](https://user-images.githubusercontent.com/83803414/168998972-bb6edec5-eeb8-4c93-9289-b8d80a26bb15.png)

### Figma design

![image](https://user-images.githubusercontent.com/83803414/168999016-06ad6a8e-e3aa-42dc-9374-1fe85a9ad129.png)

  - For project technologies, Please use the correct color `#6070FF;`
  - For model buttons, Please use the correct color `#396DF2;`

### Your design

![image](https://user-images.githubusercontent.com/83803414/168999756-88e4421c-49d7-4b14-8e4a-7ff0f4b6b756.png)

### Figma design

![image](https://user-images.githubusercontent.com/83803414/168999834-dee8e093-d30c-4ab1-af2e-6949135a4b00.png)